dataflowAPI/h/slicing.h

```diff
@@ -136,12 +136,27 @@ class Slicer {
  public:
   typedef std::pair<InstructionAPI::Instruction, Address> InsnInstance;
   typedef std::vector<InsnInstance> InsnVec;
+  typedef std::map<ParseAPI::Block *, InsnVec> InsnCache;
```

Comment:
While I believe the blocks are guaranteed not to be deleted/reused, I will refactor it to use block start address as the key to avoid any such potential danger.